uint32 max_vel_fact
uint32 max_acc_fact
---
int32 reply
